README.md

```diff
@@ -56,10 +56,6 @@
 - **[Use of Webpack configuration]**
 - **[Use of professional documentation]**
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- <a href="https://vangelif.github.io/Leaderboard/">Check out the live demo</a>
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
```

Comment:
- [ ] [OPTIOAL]
The spacings between each section of the ReadMe are so narrow that it makes the ReadMe appears quite cluttered. Let's improve upon this by allowing appropriate spacing (You can use the default spacing that comes with the original ReadMe.)